### PR TITLE
[#717] Add info about search results sort order

### PR DESCRIPTION
**Pull request by georgiana-b**
Fixes opentrials/opentrials#717

Simple writing was easy to miss so I styled it a bit:

search-info-box

### CODE REVIEW

**Comment by nightsh** [comment]
I would go for a normal weight, italic text (paragraph) followed by a `<hr>`. Harder to miss, less colourful.

**Comment by benmeg** [comment]
> I would go for a normal weight, italic text (paragraph) followed by a `<hr>`. Harder to miss, less colourful.

+1 on this

File: assets/styles/_search-results.scss [hunk 1]
```diff
@@ -69,6 +69,12 @@
     margin: 0;
     list-style: none;
   }
+
+  .info {
```
Comment (vitorbaptista) [inline assets/styles/_search-results.scss]: The `info` name is too generic IMHO, so I would prefer something like `search-results-sort-order` or similar.

**Comment by vitorbaptista** [comment]
Agreed with @nightsh.

**Comment by georgiana-b** [comment]
How it looks after applying suggestions :point_up::

spectacle t16864

**Pull request merged**